sound/soc/codecs/max98363.c

```diff
@@ -185,7 +185,8 @@ static int max98363_io_init(struct sdw_slave *slave)
 		/* make sure the device does not suspend immediately */
 		pm_runtime_mark_last_busy(dev);
 
-		pm_runtime_enable(dev);
+		if (!pm_runtime_enabled(dev))
```

Comment:
@yongzhi1 this looks a bit fishy with the runtime PM enable sequence and first_hw_init usage here.

Reply:
Yes, I agree, if pm_runtime_enable() is called by the codec driver, should there be a matching pm_runtime_disable() called in the same driver?

Reply:
@yongzhi1 it's not an issue with pm_runtime_disable() but rather the fact that this entire block is gated by the test on first_hw_init. I don't see how it's possible to execute this block twice.

Reply:
Indeed, the above line is called once, and I can only repro once upon reboot with S0ix test "suspend_stress_test -c1"
```
[   79.258335] max98363 sdw:2:019f:8363:00:1: Bus clash detected before INT mask is enabled
[   79.258589] max98363 sdw:2:019f:8363:00:1: Unbalanced pm_runtime_enable!
[   79.258840] max98363 sdw:2:019f:8363:00:0: Bus clash detected before INT mask is enabled
[   79.259113] max98363 sdw:2:019f:8363:00:0: Unbalanced pm_runtime_enable!
[   90.705383] OOM killer enabled.
[   90.708888] Restarting tasks ..
```
Will re-test with https://github.com/thesofproject/linux/pull/4345 and close this one.